internal/analyse/analyse.go

```diff
@@ -60,11 +60,36 @@ func downloadMozillaRemovedCACertsList() ([]removedCertificate, error) {
 
 	removedCerts := make([]removedCertificate, len(csvLines))
 	for i, csvLine := range csvLines {
-		removedCerts[i] = removedCertificate{
-			// From the CSV format that Mozilla publishes, the 8th column (id 7) is the fingerprint and the 23rd column
-			// (id 22) is the comment.
-			Fingerprint: csvLine[7],
-			Comments:    csvLine[22],
+		// Skip the header row
+		if i == 0 {
+			continue
+		}
+
+		// Find column indices by their names from the header
+		var fingerprint, comments int
+		if i == 1 {
+			fingerprint = -1
+			comments = -1
+			for idx, header := range csvLines[0] {
+				switch header {
+				case "SHA-256 Fingerprint":
+					fingerprint = idx
+				case "Comments":
+					comments = idx
+				}
+			}
+
+			if fingerprint == -1 {
+				return nil, fmt.Errorf("required column 'SHA-256 Fingerprint' not found in CSV header")
+			}
+			if comments == -1 {
+				return nil, fmt.Errorf("required column 'Comments' not found in CSV header")
+			}
+		}
```

Comment:
Wouldn't this make more sense being done outside of the loop? as in - only once - as validation of the headers? 

You could move over to the "Read" and only load in the first row (headers) check that, then the main logic of fingerprint/comment indexes is a little cleaner?

nit: use constants for the case statements so it's easier to update in the future?